src/OnboardingSPA/components/App/index.js

```diff
@@ -223,12 +226,57 @@ const App = () => {
 		}
 	}
 
+	const handlePreviousStepTracking = () => {
+		const previousStep = window.nfdOnboarding?.previousStepID;
+		if ( typeof previousStep !== 'string' ) {
+			window.nfdOnboarding.previousStepID = location.pathname;
+			HiiveAnalytics.dispatchEvents();
+			return;
+		}
+
+		if ( previousStep.includes( 'products' ) ) {
+			trackHiiveEvent( 'products-info', {
+				productCount:
+					currentData.storeDetails.productInfo.product_count,
+				productTypes:
+					currentData.storeDetails.productInfo.product_types.join(
+						','
+					),
+			} );
+		}
+
+		if ( previousStep.includes( 'site-pages' ) ) {
+			currentData.data.sitePages?.other?.forEach( ( sitePage ) => {
```

Comment:
Same as above step, don't we need to track every click ? and things like does the user even use, description button